docs/FAQs.md

```diff
@@ -1,101 +1,44 @@
 # FAQs
 
-## About
+## Overview
 
-### **What is the history of SkySQL?**
```

Comment:
@namanshah76 - I think we should preserve this question. Amongst developers our pedigree provides credibility. Not some new startup launching its own new DB.